modules/rest-util/src/blaze/middleware/fhir/resource.clj

```diff
@@ -95,6 +99,25 @@
       (assoc request :body resource))
     (ba/incorrect "Missing HTTP body.")))
 
+(def ^:private ^Base64$Encoder b64-encoder
+  (.withoutPadding (Base64/getUrlEncoder)))
```

Comment:
why URL encoder and why without padding? The encoder has to fit the `(.decode (Base64/getDecoder) ...)` from the output. Does FHIR say something about Url encoding and padding? Are we wrong with the output?